webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -172,6 +220,14 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      elem: null,
+      checkedBlur: false,
```

Comment:
```suggestion
      blurImage: false,
```

Reply:
then we can keep this `blurImage` and it's not as confusing
`blur` is a verb, so the user is doing and action, so the naming fits better